Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-formrecognizer/README.md

```

##### Sample code

> The following sample lets you read the scanned copy of a sample receipt. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/overview)

Reviewer: ```suggestion
The following sample reads the scanned copy of a sample receipt. To know more, refer to the [Azure Documentation on Form Recognizer](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/overview)
```
